src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -676,12 +939,35 @@ private static XmlReader CreateXmlReader(Stream stream)
             return XmlReader.Create(textReader, InternalDeserializer.XmlReaderSettingsForCliXml);
         }
 
+        private static XmlReader CreateXmlReader(StringBuilder stringBuilder)
```

Comment:
Duplicate code?

Reply:
@iSazonov , I tired to use the same function but I was not sure how to do it so I have created a overloaded the method. This is to make sure that Import-Clixml and ConvertFrom-Clixml use the same helper function.

Reply:
It is static. So it could be moved to a temp class or a base class.

Reply:
class ImportXmlHelper does't not derive from any other class.
I tried to convert sting into memory stream so that we dont need this function itself. Let me know if we can implementation this

```        
internal ImportXmlHelper(PSCmdlet cmdlet, string cliObject)
        {
            _cmdlet = cmdlet;
            _memoryStream = new MemoryStream();
            StreamWriter memeoryWriter = new StreamWriter(_memoryStream);
            memeoryWriter.Write(cliObject);
            memeoryWriter.Flush();
            _memoryStream.Position = 0;
            CreateStream();
        }
```

Reply:
It depends on whether we want to process "#< CLIXML".

Reply:
Comands like below produce output with "#< CLIXML"
```
Start-Process -FilePath $PSHOME\pwsh -RedirectStandardOutput "temp.xml" -Args "-noprofile -nologo -outputformat xml -command get-command import-clixml" -Wait
```
I have not sure where else this behavior exists.
![image](https://user-images.githubusercontent.com/30629012/83768997-f2647400-a69c-11ea-98b5-2bd00067c126.png)